Added KeyboardControllerOsc

lassse opened the pull request.

genekogan commented:
thanks!

genekogan commented:
only thing missing is the settings file. you can force commit it to bin/data. note i've renamed/simplified the name and moved it to osc-modules.

lassse commented:
Think it should be added now, also updated the readme :)

On Fri, May 19, 2017 at 5:27 PM Gene Kogan <notifications@github.com> wrote:

> only thing missing is the settings file. you can force commit it to
> bin/data. note i've renamed/simplified the name and moved it to osc-modules.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ml4a/ml4a-ofx/pull/15#issuecomment-302734209>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AAMxnTHmFQekTdhKD6ZwmS2pp3sb0Scrks5r7bT9gaJpZM4Ngdac>
> .
>
-- 
Lasse Korsgaard
http://lassekorsgaard.com
+45 26 83 81 66
